tools/assetpipelineeditor/animationwidget.cpp

```diff
@@ -234,7 +234,7 @@ QVariant AnimationClipModel::data(const QModelIndex &index, int role) const
     case 1:
         switch (role) {
         case Qt::DisplayRole:
-            return QString("%1s").arg(m_clips.at(index.row()).duration, 0, 'f', 2);
+            return QString("%1s").arg(static_cast<double>(m_clips.at(index.row()).duration), 0, 'f', 2);
```

Comment:
maybe minor (and mostly asking for the sake of my personal knowledge) but isn't QStringLiteral recommended for these strings nowadays ?

Reply:
true